### DISCUSSION TITLE
Strange page on FlyPad

### DISCUSSION THREAD

**Discussion by normie555** [flybywiresim/aircraft#4618] [Help] [answered]
![carto](https://user-images.githubusercontent.com/71556913/116247180-71ccbd00-a71f-11eb-94a4-25c953f6970a.png)
This has appeared on my FlyPad. 
What is it and how do I get rid of it

**Comment by ExampleWasTaken** [comment, marked as answer]
This is the copyright page of the map provider. Because of copyright reasons we have to include it. There is no way to get rid of it other than reloading the aircraft.

**Reply by normie555** [reply]
Thanks
